README.md

```diff
@@ -0,0 +1,40 @@
+### Advanced Building Blocks - Bubble Sort
+
+We executed the bubble sort algorithm to sort elements in these two projects.
+We used the 'yield' keyworkd and passed a block to a method for the second challenge as per the instructions of the project.
+
+## Built With
+
+- Ruby
+
+## Pre-Request
+
+Ruby installed.
+
+## Getting Started
+
+To get a local copy:
+
+- Cloning repository:
+
+  1. Press the green button "Code" and copy the link.
+  2. Open terminal or Git Bash in a folder where you want to clone the repository.
+  3. Type command "git clone" and paste the link that you copied and press "Enter"
```

Comment:
Instead of writing a whole English statement, wouldn't be better to show it?
```
$ git clone your_repo_link
$ cd folder_name
$ ruby example.rb
```
Like this.